perun-openapi/openapi.yml

```diff
@@ -8699,19 +8733,228 @@ paths:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO setSendingEnabled
+  /urlinjsonout/registrarManager/copyMails/groupToGroup:
+    post:
+      tags:
+        - RegistrarManager
+      operationId: copyMailsFromGroupToGroup
+      summary: Copy all e-mail notifications from selected Group into another.
+      parameters:
+        - { name: fromGroup, schema: { type: integer }, in: query, description: "source group" }
+        - { name: toGroup, schema: { type: integer }, in: query, description: "destination group" }
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO copyMails
+  /urlinjsonout/registrarManager/deleteApplicationMail/v:
+    post:
+      tags:
+        - RegistrarManager
+      operationId: deleteApplicationMailForVo
+      summary: Deletes an e-mail notification from DB based on id property.
+      parameters:
+        - $ref: '#/components/parameters/voId'
+        - $ref: '#/components/parameters/id'
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO deleteApplicationMail
+  /urlinjsonout/registrarManager/deleteApplicationMail/g:
+    post:
+      tags:
+        - RegistrarManager
+      operationId: deleteApplicationMailForGroup
+      summary: Deletes an e-mail notification from DB based on id property.
+      parameters:
+        - $ref: '#/components/parameters/groupId'
+        - $ref: '#/components/parameters/id'
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO updateApplicationMail
+  /json/registrarManager/updateApplicationMail:
+    post:
+      tags:
+        - RegistrarManager
+      operationId: updateApplicationMail
+      summary: Updates an e-mail notification.
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: InputUpdateApplicationMail
+              description: "input to updateApplicationMail"
+              type: object
+              required:
+                - mail
+              properties:
+                mail: { $ref: '#/components/schemas/ApplicationMail' }
 
-  #TODO addApplicationMail
+  /json/registrarManager/addApplicationMail/v:
+    post:
+      tags:
+        - RegistrarManager
+      operationId: addApplicationMailForVo
+      summary: Add new mail notification.
+      responses:
+        '200':
+          $ref: '#/components/responses/ApplicationMailResponse'
```

Comment:
The response type of this method is an integer. The documentation is actually wrong, I will fix it.